lib/index.js

```diff
@@ -43,4 +43,5 @@ ZAFClient.init = function(callback, loc) {
   return client;
 };
 
+window.ZAFClient = ZAFClient;
```

Comment:
Would it be possible to add a unit test for this functionality?

Reply:
I removed this in the next PR.